resources/views/main.blade.php

```diff
@@ -131,31 +63,16 @@
                 <img src="" alt="" id="selectedImage">
             </figure>
             <div id="uploadProgress"></div>
-            <form id="formCreatePost" action="#" autocomplete="off">
-                <input type="text" name="imageCaption" id="imageCaption" placeholder="Insert caption here..."
+            <form id="formCreatePost" method="post" action="{{ route('upload-post') }}" autocomplete="off" enctype="multipart/form-data">
+                @csrf
+                <input type="text" name="caption" id="imageCaption" placeholder="Insert caption here..."
                     required>
+                <input type="file" id="imageFileInput" accept="image/jpeg" name="image" style="display: none;">
```

Comment:
I think we can improve it by showing an error for the caption and especially the image input file here, what do you think mas @riandyrn ? 😃

Reply:
Ok, I just see this message, @farulivan. 😅🙏🏻

I will make adjustments to the blade template.

Reply:
@farulivan, I've tried to make adjustments just like you suggest. However, it was too difficult for me.

So what I'm doing right now is to show the error message in the main body like this:

![Screenshot 2024-05-06 at 22 14 33](https://github.com/farulivan/jepret/assets/5761331/3b3bc44f-6b49-4c9a-a3d6-bc5e9872d11d)

Maybe you can improve the UI later. But for now, at least, it solves the main problem: showing an error when the create post validation fails.

Reply:
Nice!! mas @riandyrn it solve the problem 🚀 already recheck and test it 😃